collagraph/__init__.py

```diff
@@ -560,14 +560,14 @@ def equivalent_code(a, b):
     In order to determine this, a number of properties are compared that
     should be equal between similar functions.
     It checks all co_* props of __code__ (as seen in Python 3.9), except for:
-      * co_varnames
       * co_firstlineno
       * co_lnotab
       * co_name
     because those vars can differ without having any impact on the equivalency
     of the functions themselves.
     """
```

Comment:
I would recommend moving `equivalent_functions` and `equivalent_code` to a separate module, it's a pretty complicated beast and it's not related to the class it is in right now.